### PR TITLE
Add DevOps Day 2016 Brasilia

### PR DESCRIPTION
**Pull request by adrianovieira**
Add our DevOps day in 2016 at Brasília/DF, Brazil.

### CODE REVIEW

**Comment by kmugrage** [comment]
New event, so calling @bridgetkromhout

**Comment by bridgetkromhout** [comment]
Thanks! Yes, we need to have a kickoff call. Let's follow up in email.

**Comment by adrianovieira** [comment]
One member of our local already did that.

Let me ask him.

**Comment by gutocarvalho** [comment]
Yes, I touch base in August 2 and sent DevOpsDays Brasilia Proposal, both in DevOpsDays Google Group and info@devopsdays.org following the wiki instructions.

We'll wait the invite for the Kickoff call.

Thanks

File: data/events/2016-brasilia.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+friendly: "2016-brasilia" # Four digit year and the city name in lower-case. Don't forget the dash!
+status: "current" # Options are "past" or "current". Use "current" for upcoming.
+
+# All dates are in unquoted YYYY-MM-DD, like this:   variable: 2016-01-05
+startdate: 2016-11-19 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2016-11-19 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: 2016-09-01 # start accepting talk proposals.
+cfp_date_end: 2016-10-15 # close your call for proposals.
+cfp_date_announce: 2016-11-15 # inform proposers of status
```
Comment (bridgetkromhout) [inline data/events/2016-brasilia.yml]: Are you really going to inform speakers they are speaking only four days before the conference? That seems way too short. Probably you should set the cfp_date_start to now, the end to at least 7 weeks out, and the announcement to at least 6 weeks out.

File: data/events/2016-brasilia.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+team_members: # Name is the only required field for team members.
+  - name: "Guto Carvalho"
+    employer: "Instruct"
+    twitter: "gutocarvalho"
+  - name: "Lauro Silveira"
+  - name: "Taciano Tres"
+  - name: "Aline Hubner"
+  - name: "Dirceu Silva"
+  - name: "Rafael Gomes"
+  - name: "Eustáquio Guimaraes"
```
Comment (bridgetkromhout) [inline data/events/2016-brasilia.yml]: `Guimarães` according to your email.

File: data/events/2016-brasilia.yml (hunk 1 above)
Comment (gutocarvalho) [inline data/events/2016-brasilia.yml]: Ok, We'll fix this, thanks!

File: data/events/2016-brasilia.yml (hunk 2 above)
Comment (gutocarvalho) [inline data/events/2016-brasilia.yml]: Typo, thanks!

**Comment by bridgetkromhout** [comment]
This can be merged after those couple of fixes.

**Comment by mattstratton** [comment]
You still have Arrested DevOps as a Community Sponsor. Which is totally fine, as long as you want that to be the case. 

If so (and we at ADO think this is great), just make sure to create the appropriate discount code (`ADO2016` for 20% off) when you set up tickets ;)

**Comment by gutocarvalho** [comment]
Discount code created

**Pull request merged**